official/resnet/resnet_run_loop.py

```diff
@@ -370,6 +393,46 @@ def _dense_grad_filter(gvs):
       eval_metric_ops=metrics)
 
 
+def set_environment_vars(flags_obj):
```

Comment:
The function name here sounds weird - it is doing more than setting environment variables and is returning a `ConfigProto`. Add some documentation? And perhaps call this:

```python
def set_environ_and_config(flags_obj):
  """Sets environment variables and returns a tf.ConfigProto for the tf.Session based on flags"""

Reply:
Done.